### ISSUE TITLE
Broken on 0.2 and master

### ISSUE THREAD

**Issue by essenciary** [codeneomatrix/Faker.jl#14]
It worked well in the past but now it fails: 

```julia
julia> using Faker
[ Info: Precompiling Faker [0efc519c-db33-5916-ab87-703215c3906f]
ERROR: LoadError: IOError: chdir ../src/locales/en: no such file or directory (ENOENT)
Stacktrace:
 [1] uv_error at ./libuv.jl:85 [inlined]
 [2] cd(::String) at ./file.jl:76
 [3] loadfiles(::String) at /Users/adrian/.julia/packages/Faker/mcRew/src/Faker.jl:27
 [4] top-level scope at none:0
 [5] include at ./boot.jl:317 [inlined]
 [6] include_relative(::Module, ::String) at ./loading.jl:1041
 [7] include(::Module, ::String) at ./sysimg.jl:29
 [8] top-level scope at none:2
 [9] eval at ./boot.jl:319 [inlined]
 [10] eval(::Expr) at ./client.jl:389
 [11] top-level scope at ./none:3
in expression starting at /Users/adrian/.julia/packages/Faker/mcRew/src/Faker.jl:50
ERROR: Failed to precompile Faker [0efc519c-db33-5916-ab87-703215c3906f] to /Users/adrian/.julia/compiled/v1.0/Faker/wbANP.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1187
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:855
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:837
```

**Comment by codeneomatrix** [comment]
thanks, for notifying this error.
please share the version of julia you are using, to be able to work on a solution

**Comment by essenciary** [comment]
Sorry, I forgot: 

```julia
julia> versioninfo()
Julia Version 1.0.1
Commit 0d713926f8 (2018-09-29 19:05 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)
```

**Comment by codeneomatrix** [comment]
thank you very much, in the next few days it will be solved

**Comment by codeneomatrix** [comment]
good day, the reported error has been fixed in version 0.2.1
https://github.com/codeneomatrix/Faker.jl/releases/tag/v0.2.1

Thank you very much, for your contribution.

**Comment by essenciary** [comment]
Yeee, thank you!
